static/javascripts/components/survey_card.jsx

```diff
@@ -2,6 +2,7 @@
 /*
 * Survey. Filled with data to send (JSON)
 * survey_handler.py/def _survey_from_request(self):
+* models/question.py
 */
 var survey = [
```

Comment:
I cannot coment the code since is merged, but I'm planning to create different nodes for each question (line 63). Inside each question, there would be different nodes for each option (line 74, not done yet). Not sure if I have to fill the initial variable in real time or just when the person clicks send survey. I was thinking doing it with javascript (append or push) when the user clicks send. Then send with AJAX. The code is messy, it wasn't supposed to be merged. Let me know if I am doing it right and what you think. @wannabeCitizen